adr/002-go.md

```diff
@@ -0,0 +1,18 @@
+# 2. Go
+
+Date: 2024-05-14
+
+## Decision
+
+We will use Golang for the SFTP service.
```

Comment:
Pedantic - I think 'Go' is the official name, just Golang is easier to search for. Maybe switch to 'Go' here and on line 16 but also add a link for clarity? **https://go.dev/**

Reply:
Fixed, thanks for pointing this out